docs/Projects/TAD/ADRs/0002-tad-reporting-standard.md

```diff
@@ -0,0 +1,51 @@
+# TAD-0002 TAD Reporting Standard
```

Comment:
I would add the TAD ADR in the TAD repo. Else we get a mess in all the ADRs and where they are.

Reply:
We don't really have a "TAD repo", or do we?

Reply:
We do not but i would not add all ADRs in this repo. Would prefer to keep them as close as possible to the project.

Reply:
Decided to keep the current structure for now.